src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -84,6 +85,11 @@ export class OffsetablePieceComponent {
 
   @HostBinding('class')
   public get getPieceTypeModifierClass(): string {
-    return (this.piece as Tv2Piece).metadata.type.toLowerCase().replace(/_/g, '-')
+    const piece: Tv2Piece = this.piece as Tv2Piece
+    const pieceTypeModifierClass: string = piece.metadata.type.toLowerCase().replace(/_/g, '-')
+    if (piece.metadata.audioMode && piece.metadata.audioMode === Tv2AudioMode.VOICE_OVER) {
+      return pieceTypeModifierClass + '-voice-over'
```

Comment:
Magic string. Would be nice to atleast have it as a local variable.

Reply:
This no longer exists!